packages/cli/tsconfig.build.json

```diff
@@ -11,6 +11,7 @@
     "types": ["node", "jest"],
     "outDir": "dist",
     "skipLibCheck": true,
+    "baseUrl": ".",
```

Comment:
Why is this needed out of interest?

Reply:
This is needed in order to properly use the tsconfig-paths: 
![image](https://github.com/Budibase/budibase/assets/15987277/7acfabf9-633c-4e66-99aa-6ffdbdd075ca)